documentation/debugging_CLI_attach_to_process_method.asciidoc

```diff
@@ -0,0 +1,81 @@
+= Debugging the CLI Attach to Process Method in CobiGen
+
+Debugging the CLI Attach to Process Method in CobiGen involves attaching a debugger to a running process of the CobiGen command line interface (CLI). This allows you to inspect the state of the process and see what is causing an issue or error.
+
+
+===== Prerequisites
+
+Before we begin, make sure that you have the following:
+
+- A working knowledge of Java and the command-line interface.
+- A development environment set up for CobiGen, including all the required tools and libraries.
+
+
+Here are the steps to debug the CLI attach to process method in CobiGen:
```

Comment:
Maybe add another section title above or instead of this line, so the prerequisites and the actual tutorial are clearly separated. Something like "Steps to start debugging".